### DISCUSSION TITLE
Full documentation on Sizif 512

### DISCUSSION THREAD

**Discussion by oO-cozy-Oo** [UzixLS/zx-sizif-512#19] [General]
I would love a full Documentation on this great Board. The pin jumper setting of what they do and all the meanings to all the settings, it is a shame that so much time has been put into this project and so little paperwork for it. I have been looking for a forum dedicated for this board but there`s no full info on it.
some Info
The power connector is a Type B: 6.0 mm OD, 2.1 mm ID. I use a 12V PSU from an external HDD unit. On a bench PSU the Sizif registers around 300mA @ 12V. You can use a DC supply of any polarity of between 9 and 12 volts.

The CPLD and ROM are programmed with the latest versions. So depending on the Jumper settings you get the ZX spectrum, ZX81 and two different test ROMs.

If you want to use the save settings feature e.g. so that it starts in 128K mode rather than Pentagon you will need to have a jumper at JP4. Do not power down the Sizif whilst it writes the configuration though as you could corrupt the ROM. It is not recommended to leave JP4 permanently jumpered as a rogue program could conceivably corrupt the ROM.

If you want to try out the built in test ROM don't have the SD card plugged in at the same time.

To use the Wifi first navigate to the 'Network' folder and run Netman (this will only work in Pentagon or 128k Modes), it will take a minute to run and sometimes throws up some errors but will eventually ask you to select your wifi Network and the password. You only have to do this the once.

You can then try it out by going to the Gopher directory and running Ugoph, If you then page down and select the first BBS you can then access spectrum games and screens to download.

Using Bluetooth is as simple as connecting to it with your phone (look for device MH-H18 or HW-BT) and playing a WAV file. It acts as if you have a cassette recorder plugged in. Please note that the latest modules (HW-BT) have an updated chipset with a builtin voice which you might hear through your speakers/TV on power up or when connecting to the module.
The RTC.SYS error on boot up with ESXDOS is expected behaviour, it is a driver for Spectrums fitted with a Real Time Clock, the Sizif does not have one.

Not all software will work on all versions (much like the real thing) for example Atic Atac will only run in 48k mode, the Across demo looks weird in anything but Pentagon mode.

The LED on the WiFi module only flashes when it sends data so most of the time it will be off.

DivMMC is not compatible with 128k Basic which is why in 128k mode with an SD card inserted it will boot into what looks like 48k mode. DivMMC is compatible with the 128k at a hardware level so most software will not have a problem running. If you want to manually type commands in like the 128k mode rather than single keypresses you can use the Magic button menu to select ‘LG’ for 48k ROM setting.

:(

**Comment by UzixLS** [comment]
@oO-cozy-Oo thanks for your work!

> Do not power down the Sizif whilst it writes the configuration though as you could corrupt the ROM.

No. In worst case you might loose your configuration and nothing more.

> It is not recommended to leave JP4 permanently jumpered as a rogue program could conceivably corrupt the ROM.

Flash IC requires special write procedure which accessible only by magic ROM. Any other spectrum software have no any ability to write flash on Sizif.
But that's not true if you decided to not follow board documentation and build your Sizif with EEPROM IC instead of Flash. EEPROM may be programmed just by asserting write operation, without following complex protocol like for Flash.

> If you want to try out the built in test ROM don't have the SD card plugged in at the same time.

That was fixed long time ago, you don't need to eject SD with latest firmware.

**Comment by oO-cozy-Oo** [comment]
@UzixLS Thank you for the reply 👍 

Have you any more Info on the sizif like for jumper 2,3, and 4 ?

I`ve got mine into a zx 48+ case some info and pictures here, I will put all the info that we get onto this site.

https://spectrumcomputing.co.uk/forums/viewtopic.php?t=7110

**Reply by UzixLS** [reply]
> Have you any more Info on the sizif like for jumper 2,3, and 4 ?

I'm not sure what sort of information needed, JP2/JP3 forces ROM address lines A18/A17 to 1, JP4 connect rom WriteEnable input to the CPU, that's it.

> I`ve got mine into a zx 48+ case some info and pictures here, I will put all the info that we get onto this site.

Looks pretty nice :)

**Reply by oO-cozy-Oo** [reply]
Thank you for the jp setting I did not know what the others was used for Thanks

**Comment by oO-cozy-Oo** [comment]
While I was messing with the jumper setting a menu came up with options to select ZX80 ZX81 HE Gluk ???? the back ground had messed up square flashing boxes any ideas , I may have held a key down while i was powering up the Sizif?

**Reply by UzixLS** [reply]
Hold CS key on poweron for boot menu. 
(HE)GLUK requires a BDI addon to work.

**Comment by oO-cozy-Oo** [comment]
Thank you for the reply :) this board is amazing you have done a great job!!!!

**Comment by BochoySNR** [comment]
Hello. Any ideas where I could buy replacement magic/reset buttons for this incredible board,the plastic buttons have broken off but still work with a pent-up.  I'll probably have to solder the new ones in. Thanks

**Comment by LuckStilon** [comment]
Together with @tuFFo126, we have written instructions for Sizif. 52 pages book, but for now only in Polish. It is possible that it will be available in English soon.

![obraz](https://github.com/user-attachments/assets/c3ac1a31-5ea9-45f7-bf9b-0b5abcb2860d)
![obraz](https://github.com/user-attachments/assets/4522b6bb-fe6e-4031-90af-e7be8dce7f44)

**Reply by XyloZX** [reply]
Hi, can you please share the manual? At least the Polish version. Thx

**Reply by tuFFo126** [reply]
instructions are available here:
https://shop.mumio.dev/Instrukcja-ZX-Spectrum-Neo-Sizif-512-red-III-52-str--p61

**Reply by LuckStilon** [reply]
Instruction is not shared free as PDF. Only printed version. Now is only in Polish language, but we are working on the translation into English.
